### PR TITLE
Implement flickr tags in Pages. Fixes #3

### PR DESCRIPTION
**Pull request by gpoo**
The replacement of tags in pages is the same as in articles,
except the name of an attribute. This patch provides a helper
function that deal with both pages and articles.

This proposal is simpler than the one in #3

### CODE REVIEW

File: pelican_flickrtag/plugin.py [hunk 1]
```diff
@@ -145,11 +144,21 @@ def replace_article_tags(generator):
...
+    logger.info('[flickrtag]: Parsing articles for photo ids...')
+    total = replace_tags(generator, generator.articles)
+    logger.info('[flickrtag]: Found %d photos ids in articles' % total)
 
 
 def replace_page_tags(generator):
-    pass
+    logger.info('[flickrtag]: Parsing documents for photo ids...')
+    total = replace_tags(generator, generator.pages)
+    logger.info('[flickrtag]: Found %d photos ids in documents' % total)
```
Comment (gpoo) [inline pelican_flickrtag/plugin.py]: Oops. I meant `pages` in the text instead of `documents`, but you get the idea.

**Review by streeter — CHANGES_REQUESTED** [review]
Can you rebase to fix the merge conflict?

**Comment by gpoo** [comment]
Done,

**Review by streeter — APPROVED** [review]

**Comment by streeter** [comment]
Thanks @gpoo! 🎉 I'll get a new release out to PyPI

**Pull request merged**